packages/augur-tools/src/templates-source.ts

```diff
@@ -9772,6 +9773,43 @@ export const TEMPLATES = {
           },
         ],
       },
+      [DEFI]: {
+        templates: [
+          {
+            marketType: SCALAR,
+            question: `What will the Total Value Locked "USD" in Defi be on [0] (UTC), according to https://data.defipulse.com/ ?`,
+            example: `What will the Total Value Locked "USD" in Defi be on 03/27/2022 (UTC), according to https://data.defipulse.com/ ?`,
+            denomination: 'Price',
```

Comment:
For scalar markets, shouldn't this be "Total Value Locked"?

Reply:
Done!